armi/reactor/tests/test_components.py

```diff
@@ -352,6 +429,13 @@ def test_getBoundingCircleOuterDiameter(self):
 
 
 class TestCircle(TestShapedComponent):
+    """Test circle shaped component.
+
+    .. test:: Circle shaped component
+        :id: T_ARMI_COMP_SHAPES0
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Delete this one.

What we found out yesterday was we can NOT put test crumbs at the class-level, they only work at the test-level.

So, not your fault. That is just a recent discovery.

And we will have to fix that elsewhere in ARMI, I think.